app/templates/head.hbs

```diff
@@ -1,4 +1,4 @@
-<title>{{model.title}}</title>
-<meta property="st:title" content={{model.title}}>
-<meta property="og:title" content={{model.title}}>
-<meta name="twitter:title" content={{model.title}}>
\ No newline at end of file
+<title>{{this.model.title}}</title>
```

Comment:
**praise:**

Nice job with using `this.model` instead of `@model`. (I had inadvertently used the latter and caused a production bug.)